### PR TITLE
Update files for the driver vc4-kms-v3d

### PR DESCRIPTION
**Pull request by sc74**

### CODE REVIEW

File: configs/kwinrc [hunk 1]
```diff
@@ -2,5 +2,4 @@
 update_info=kwin.upd:replace-scalein-with-scale,kwin.upd:port-minimizeanimation-effect-to-js,kwin.upd:port-scale-effect-to-js,kwin.upd:port-dimscreen-effect-to-js,kwin.upd:auto-bordersize,kwin.upd:animation-speed,kwin.upd:desktop-grid-click-behavior,kwin.upd:no-swap-encourage,kwin.upd:make-translucency-effect-disabled-by-default,kwin.upd:remove-flip-switch-effect,kwin.upd:remove-cover-switch-effect,kwin.upd:remove-cubeslide-effect,kwin.upd:remove-xrender-backend,kwin.upd:enable-scale-effect-by-default
 
 [Compositing]
-Enabled=false
```
Comment (MattBlack85) [inline configs/kwinrc:5]: Why you want composition enabled by default?

File: systemd/x0vncserver.service [hunk 2]
```diff
@@ -1,6 +1,7 @@
 [Unit]
 Description=Remote desktop service (VNC) for :0 display
 Requires=display-manager.service
+After=network-online.target
```
Comment (MattBlack85) [inline systemd/x0vncserver.service:4]: this has never made a n appeareance here as it was causing very long start times when not connected to the net

File: configs/kwinrc (hunk 1 above)
Comment (sc74) [reply configs/kwinrc:5]: Without the composer, the tigervnc client-server connection is unstable. It causes the client to close

File: systemd/x0vncserver.service (hunk 2 above)
Comment (sc74) [reply systemd/x0vncserver.service:4]: Indeed, this causes a longer startup, but this is necessary for the vnc dummy server configuration to be taken into account. Without this, the geometry is not carried out by x0vncserver and the display becomes a square geometry screen
The additional time taken at the start remains acceptable compared to the gain obtained by activating all the graphics options of the driver

File: configs/kwinrc (hunk 1 above)
Comment (MattBlack85) [reply configs/kwinrc:5]: I get it, but on pi4 it may make the whole thing clunky

File: systemd/x0vncserver.service (hunk 2 above)
Comment (MattBlack85) [reply systemd/x0vncserver.service:4]: I am not sold. How the network makes this working? Network should have nothing to do with video, maybe we should find the right dependencies here?

File: configs/kwinrc (hunk 1 above)
Comment (sc74) [reply configs/kwinrc:5]: I didn't notice any problem on my pi 4. I realized these were changes on pi 4 before testing on pi 5
Did you test on your pi 4 with compositor then without?

File: systemd/x0vncserver.service (hunk 2 above)
Comment (sc74) [reply systemd/x0vncserver.service:4]: I actually think the problem is a tigervnc related issue directly, not the video server. In the wiki, they build a systemd service with After=network-online.target.
When I didn't use this line, x0vncserver displayed a square virtual screen. After adding, the configuration is taken into account by x0vncserver. I imagine that there is in tigervnc that if it does not obtain a fast enough response from the network, it launches a server without configuration with square display. As the pi becomes a little slower now with the new video driver, we encounter this problem

Comment (sc74) [reply systemd/x0vncserver.service:4]: I did not test by removing this line to see if the pi 5 which is faster encountered the same issue

File: configs/kwinrc (hunk 1 above)
Comment (MattBlack85) [reply configs/kwinrc:5]: yes, I sometimes turn it on (also for some app like collimator circle is needed) and I noticed on the pi it is clunky after a while, haven't noticed any big difference on the pi5 tho. Maybe we could generate this file on the first boot checking which model we run on?

Comment (sc74) [reply configs/kwinrc:5]: Did you test with the vc4-kms-v3d driver and file modifications or with the current configuration?

**Pull request merged**